app/assets/scripts/config/staging.js

```diff
@@ -6,5 +6,6 @@
 module.exports = {
   environment: 'staging',
   api: 'https://ytr9800fbk.execute-api.us-east-1.amazonaws.com',
+  apiv2: 'https://ytr9800fbk.execute-api.us-east-1.amazonaws.com/v2',
```

Comment:
I think we should be able to simply add /v2 to the above url and use the v2 api for all requests. That way we don't have to maintain two versions, making us wonder when to use which one.